rebar.config

```diff
@@ -61,6 +61,12 @@
   {short, [{erl_opts, [{d, 'EQC_SHORT_TEST'}]}]},
   {long, [{erl_opts, [{d, 'EQC_LONG_TEST'}]}]},
   {prod, [{relx, [{dev_mode, false}]}]},
+  {deb, 
+	  [{relx, 
+		  [{dev_mode, false},
+		  {overlay_vars, "rel/vars/deb.config"}]},
+	  {cuttlefish,
```

Comment:
If I understand correctly, you are asking why I need a separate deb config?  This is more of a consistency problem, the paths in the deb seem to be best suited to linux and may not be applicable to those on other OS's.  This is the case with DFE, and DDB where the defaults seem to be applicable in the general sense, and the deb overrides are suited to linux service conventions. I hope I have understood your question correctly.